File: PUGD-api/models/AcquisitionModels/Order_line.go
package AcquisitionModels

import "go.mongodb.org/mongo-driver/bson/primitive"

type Order_Line struct {
	ID               primitive.ObjectID `json:"_id,omitempty" bson:"_id,omitempty"`
	Order            primitive.ObjectID `json:"Order"`
	Isbn             string             `json:"isbn"`
	Title            string             `json:"title"`
	Author           string             `json:"author"`
	Quantity         int                `json:"quantity"`
	QuantityReceived int                `json:"quantityreceived"`
	QuantityFactured int                `json:"quantityfactured"`
	Price            float64            `json:"price"`
	Discount         float64            `json:"discount"`
	Status           string             `json:"status"`
}

File: PUGD-frontend/graphql/queries/acquisition/orderline.js
import gql from "graphql-tag";
const GetAllOrderLines = gql`
  query($order: String!) {
    getAllOrderLines(order: $order) {
      _id
      isbn
      title
      author
      quantity
      price
      discount
      status
    }
  }
`;

module.exports = {
  GetAllOrderLines,
};
